GliaWidgetsTests/CallVisualizer/VideoCall/VideoCallTests.swift

```diff
@@ -66,4 +66,31 @@ final class VideoCallTests: XCTestCase {
         headerProps.backButton?.tap()
         XCTAssertEqual(wasExecuted, true)
     }
+
+    func test_end_screen_sharing_visible() {
```

Comment:
We do not have written agreement, but previously, at leat during review of my PRs, there was agreement team mates to stick to conventional naming Swift style, with exception to `test_` prefix:
```suggestion
    func test_endScreenSharingIsVisible() {
```